app/controllers/topics.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+helpers do
+  def res(status_code, message)
+    status status_code
+    {
+      success: true,
+      status: status_code,
+      message: message
+    }.to_json
+  end
+end
+
+get '/topics' do
+  require_admin
+  @topics = Topic.all
+  gon.topic_titles = @topics.map(&:title)
+  @user = User.first(email: session[:email])
```

Comment:
this could be a method because it has been repeated severally
Try something like

``` ruby
def current_user(email)
  User.first(email: email)
end
```